ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo

```diff
@@ -0,0 +1,98 @@
+within N44.DAEMode_TestCases.SMIB_Examples.Example_1.Generator;
+model GenEventTest "Configuration of synchronous generator with regulators: GENSAL, HYGOV, SCRX, STAB2A;
+  Nordic 44 model: Buses 3115, 5300, 6100, 6700, 7100"
+    extends OpenIPSL.Electrical.Essentials.pfComponent;
+    OpenIPSL.Electrical.Machines.PSSE.GENSAL gENSAL(
+          Tppd0=0.05 "T''do (> 0)",
+          Tppq0=0.15 "T''qo (> 0)",
+          D=0 "Speed Damping",
+          Tpd0=6.4,
+          H=3.5,
+          Xd=1.14,
+          Xq=0.84,
+          Xpd=0.34,
+          Xppd=0.26,
+          Xppq=0.26,
+          Xl=0.2,
+          M_b=1200,
+          S10=0.1,
+          S12=0.3,
+          R_a=0,
+          V_b=V_b,
+          V_0=V_0,
+          angle_0=angle_0,
+          P_0=P_0,
+          Q_0=Q_0) annotation (Placement(transformation(extent={{-64,-24},{-2,40}})));
+    OpenIPSL.Electrical.Controls.PSSE.TG.HYGOV hYGOV(
+          R=0.06 "Permanent droop, p.u",
+          r=0.4 "Temporary droop, p.u",
+          VELM=0.2 "Gate open/close velosiy limit, p.u/sec",
```

Comment:
per-unit unit is either "pu" which I would prefer, or "p.u." but no mix.

Reply:
Since it is also about descriptions in modifiers of a model, I rather delete it instead of correcting the term per unit.

Reply:
Yes sure. No point in correcting here.